lib/ecto_stream_factory/factory.ex

```diff
@@ -71,7 +157,7 @@ defmodule EctoStreamFactory.Factory do
       apply(module, function_name, [])
     else
       raise EctoStreamFactory.UndefinedGeneratorError,
-        module: module_to_string(module),
+        module: module,
```

Comment:
This is a breaking change, isn't it? Since this is a pre-1.0 version, breaking changes are allowed in a minor release according to semver, but maybe it's worth mentioning in the release notes? But then again, exceptions are not currently documented, so you could argue users can't rely on the exception attributes anyway...

Reply:
Good point, I haven't even thought about that. But as you said, this is undocumented and I'm pretty sure not used by anyone so I prefer to skip adding such a minor detail to the release notes.